app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterDuplicateMovementReferenceNumberController.scala

```diff
@@ -46,8 +46,8 @@ class EnterDuplicateMovementReferenceNumberController @Inject() (
   val sessionStore: SessionCache,
   claimService: ClaimService,
   featureSwitch: FeatureSwitchService,
-  enterDuplicateMovementReferenceNumberPage: pages.enter_duplicate_movement_reference_number,
-  enterNoLegacyDuplicateMrn: pages.enter_no_legacy_duplicate_mrn
+  enterDuplicateMovementReferenceNumberPage: pages.enter_duplicate_mrn_or_entry_number,
```

Comment:
best to change the name of the variable to reflect the name of the template

Reply:
Done